examples/raw.rs

```diff
@@ -1,6 +1,7 @@
 // Copyright 2018 TiKV Project Authors. Licensed under Apache-2.0.
 
 #![feature(async_await, await_macro)]
+#![type_length_limit = "3081103"]
```

Comment:
This is a [long, long](https://www.youtube.com/watch?v=6-1Ue0FFrHY) type!

Reply:
What did I just watch? :-p